### ISSUE TITLE
mPower 3 Outlet subs wrong

### ISSUE THREAD

**Issue by dersch81** [magcode/mpower-tools#9]
Hi, first of all it is working very easy! Thanks for your great work!

I have 2 devices mPower Mini and 3 Port Outlet.

With the Mini i have all subs as expected, voltage, power, energy everything is fine. 

But with the 3 Port mPower i don't have any energy value it is always 0 for all ports.

**Comment by magcode** [comment]
Hi,

can you check the output of

`cat /proc/power/enabled1`

In case it returns `0` please do the following:
`echo 1 > /proc/power/enabled1`

Then try again (plug in a device on port 1 and check in MQTT)

If that works now do the same for the other ports (enabled2 ... enabled3)

**Comment by dersch81** [comment]
Yes that was the case. Now i get the energy value via MQTT. Thanks

**Comment by magcode** [comment]
Thanks for feedback. I'll either add this to the documentation or maybe add some scripting to enable it automatically.
